ansible/vars/aeternity/api-main.yml

```diff
@@ -0,0 +1,123 @@
+api_base_uri: http://{{ public_ipv4 }}:{{ node_config.http.external.port }}/v2
+configure_peers: false
+seed_peers:
+  - "18.136.37.63"
+  - "52.220.198.72"
+  - "3.16.242.93"
+  - "3.17.30.101"
+  - "13.58.177.66"
+  - "13.250.190.66"
+  - "34.218.57.207"
+  - "34.209.38.2"
+  - "18.217.69.24"
+  - "3.0.217.255"
+  - "3.17.15.122"
+  - "3.17.30.125"
+  - "35.166.231.86"
+  - "52.11.110.179"
+  - "54.214.159.45"
+  - "52.88.74.110"
+  - "3.0.221.40"
+  - "18.218.172.119"
+  - "52.40.117.141"
+  - "34.211.251.83"
+  - "13.250.144.60"
+  - "35.163.118.175"
+  - "3.0.12.164"
+  - "18.216.167.138"
+  - "3.17.15.239"
+  - "3.0.86.27"
+  - "52.26.157.37"
+  - "3.17.17.128"
+  - "13.228.202.140"
+  - "52.77.168.79"
+  - "13.53.114.199"
+  - "13.53.149.181"
+  - "13.53.164.121"
+  - "13.53.77.98"
+  - "13.53.213.137"
+  - "13.53.51.175"
+  - "13.53.161.210"
+  - "13.53.162.212"
+  - "13.53.89.32"
+  - "13.53.78.163"
+  # London (eu-west-2) - DEPRECATED
+  - "35.177.192.219"
+  - "3.8.105.183"
+  - "3.8.30.66"
+  - "35.177.165.232"
+  - "35.177.212.38"
+node_config:
+  peers:
```

Comment:
We don't need this list here, it's handled by the node for `ae_mainnet`. Make sure to remove that key and not keep it empty